### ISSUE TITLE
Implement nucleotide drawer

### ISSUE THREAD

**Issue by anderspitman** [zmertens/SeqAndDestroy#17]
The idea is that the player should be able to select what nucleotide they want to fire directly instead of having to cycle through the options. This way they don't have to memorize the order.

**Comment by zmertens** [comment]
I can't remember if this was already discussed but what about having 4 separate (so they're easier to touch) images at the bottom and then the user can choose one at a time (maybe add a refresh rate?).

**Comment by anderspitman** [comment]
Yep that's exactly what I had in mind. I think it will make it much more intuitive. I find myself getting frustrated and almost never using the switching feature because there's not enough time

**Comment by anderspitman** [comment]
I started to implement this and realized it might actually give the player too much power. I think I'll hold off for now, but this is definitely something we might want to do later.

**Comment by zmertens** [comment]
What about having the 4 nucleotides associated with a side of the screen?

```js
              A
   --------------------|
   |                        |
C |                       |    U
   |                       |
   |_____________|
              G
```

I also had another idea... What to do about indels? They are inserted deletions and deleted deletions. They could be represented as partial nucleotides. For instance, each nucleotide takes like 4 hits or something to break, and an indel is represented as < 4 hits.
